src/devices/AD5328/README.md

```diff
@@ -0,0 +1,14 @@
+# AD5328 - Digital to Analog Convertor
+AD5328 is an Digital-to-Analog converter (DAC) with 12 bits of resolution.
```

Comment:
Markdown 101: new line before and after all title #, no except for the first one (no need before)

Reply:
done